mlrun/utils/notifications/notification_pusher.py

```diff
@@ -490,6 +508,8 @@ def push_pipeline_run_results(
 def _sanitize_notification(notification: mlrun.model.Notification):
     notification_dict = notification.to_dict()
     notification_dict.pop("secret_params", None)
+    notification_dict.pop("message", None)
+    notification_dict.pop("params", None)
```

Comment:
Is it possible that the exception text (in `reason`) will contain sensitive information?

Reply:
this function is for logging purposes only. reason has nothing with sensitive information but the exception message